Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -175,8 +186,44 @@ class ARExample: UIViewController {
         
         // Hide directions view if we're calibrating.
         userDirectionsView.isHidden = startCalibrating
+        
+        // Disable changing scenes if we're calibrating.
+        sceneItem.isEnabled = !startCalibrating
     }
-
+    
+    /// Sets up the required functionality in order to display the scene and AR experience represented by `sceneInfo`.
+    ///
+    /// - Parameter sceneInfo: The sceneInfo used to set up the scene and AR experience.
+    fileprivate func selectSceneInfo(_ sceneInfo: SceneInfoType?) {
+        guard let info = sceneInfo else { return }
+        
+        // Set currentSceneInfo to the selected scene info.
+        self.currentSceneInfo = info
```

Comment:
Now that this is no longer inside a closure, you could get rid of all the explicit references to `self`, if you want to.